packages/dds/merge-tree/src/properties.ts

```diff
@@ -69,10 +69,10 @@ export function matchProperties(
  */
 export function extend<T>(base: MapLike<T>, extension: MapLike<T> | undefined): MapLike<T> {
 	if (extension !== undefined) {
-		for (const [key, v] of Object.entries(extension)) {
-			if (v === undefined) {
-				continue;
-			} else if (v === null) {
+		// eslint-disable-next-line guard-for-in, no-restricted-syntax
```

Comment:
this looks like a functional change which is incorrect.

Reply:
ah thank you for the catch, reverted this